sphinx_gallery/downloads.py

```diff
@@ -117,14 +117,14 @@ def generate_zipfiles(gallery_dir):
             target_dir = os.path.join(gallery_dir, directory)
             listdir.extend(list_downloadable_sources(target_dir))
 
-    py_zipfile = python_zip(listdir, gallery_dir)
-    jy_zipfile = python_zip(listdir, gallery_dir, ".ipynb")
+    py_zipfile = os.path.basename(python_zip(listdir, gallery_dir))
+    jy_zipfile = os.path.basename(python_zip(listdir, gallery_dir, ".ipynb"))
```

Comment:
Probably better to just modify `python_zip` to do the right thing. I'll see if I can fix it, as `os.path.basename` is linking to the wrong place currently (just `doc/` / the root, which won't be correct). Probably needs `gallery_dir/` prepended...